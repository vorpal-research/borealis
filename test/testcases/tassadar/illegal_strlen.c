#include <stdlib.h>
#include <string.h>

void borealis_assert(int);
void borealis_assume(int);
int borealis_nondet();

#define ASSERT(X) borealis_assert((int)(X))
#define ASSUME(X) borealis_assume((int)(X))
#define __NONDET__ borealis_nondet

int main() {

	char* mes = malloc(1);

	ASSERT(strlen(mes) == 666);

	free(mes);

}