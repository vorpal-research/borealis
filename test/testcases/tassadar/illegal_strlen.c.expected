[
"Memory read violation at 0x800100000"
]